data_steward/tools/generate_ext_tables.py

```diff
@@ -171,17 +170,21 @@ def create_and_populate_source_mapping_table(project_id, dataset_id):
     return rows_affected
 
 
-def get_generate_ext_table_queries(project_id, deid_dataset_id,
+def get_generate_ext_table_queries(project_id, dataset_id, sandbox_dataset_id,
                                    mapping_dataset_id):
     """
     Generate the queries for generating the ext tables
     :param project_id: project_id containing the dataset to generate ext tables in
-    :param deid_dataset_id: deid_dataset_id to generate ext tables in
-    :param mapping_dataset_id: mapping_dataset_id to use the mapping tables from
+    :param dataset_id: dataset_id to generate ext tables in
+    :param sandbox_dataset_id: sandbox_dataset_id to store sandboxed rows.
+    :param mapping_dataset_id: mapping_tables_dataset_id to use the mapping tables from
     :return: list of query dicts
     """
     queries = []
 
+    # FIXME: Remove ths reference in future
+    logging.info(f'sandbox_dataset_id : {sandbox_dataset_id}')
```

Comment:
Minor suggestion:
```suggestion
    LOGGER.info(f'sandbox_dataset_id : {sandbox_dataset_id}')
```
nit: in line 9 of this file there is a declaration of LOGGER that can be removed if you chooose.